<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MMainWindow</class>
 <widget class="QMainWindow" name="MMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>699</width>
    <height>449</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MMainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QListView" name="listView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>221</width>
      <height>311</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="material_list_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;b&gt;Materials</string>
    </property>
    <property name="textFormat">
     <enum>Qt::RichText</enum>
    </property>
   </widget>
   <widget class="QLabel" name="collision_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Collision&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="textFormat">
     <enum>Qt::RichText</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="collision_button">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>141</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Choose Collision</string>
    </property>
   </widget>
   <widget class="QLabel" name="material_name">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>110</y>
      <width>451</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;No material selected&lt;/span&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="textFormat">
     <enum>Qt::RichText</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>130</y>
      <width>141</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Choose Material</string>
    </property>
   </widget>
   <widget class="QLabel" name="file_name_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>691</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;No file loaded&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="textFormat">
     <enum>Qt::RichText</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>699</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave_File"/>
    <addaction name="actionSave_Animation"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>Open File...</string>
   </property>
  </action>
  <action name="actionSave_File">
   <property name="text">
    <string>Save File...</string>
   </property>
  </action>
  <action name="actionSave_Animation">
   <property name="text">
    <string>Save Animation...</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>actionOpen</sender>
   <signal>triggered()</signal>
   <receiver>MMainWindow</receiver>
   <slot>openNewFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>349</x>
     <y>224</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSave_File</sender>
   <signal>triggered()</signal>
   <receiver>MMainWindow</receiver>
   <slot>saveFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>355</x>
     <y>224</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>listView</sender>
   <signal>clicked(QModelIndex)</signal>
   <receiver>MMainWindow</receiver>
   <slot>selectMaterial(QModelIndex)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>120</x>
     <y>272</y>
    </hint>
    <hint type="destinationlabel">
     <x>355</x>
     <y>224</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>MMainWindow</receiver>
   <slot>chooseMaterial()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>300</x>
     <y>137</y>
    </hint>
    <hint type="destinationlabel">
     <x>355</x>
     <y>224</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>collision_button</sender>
   <signal>clicked()</signal>
   <receiver>MMainWindow</receiver>
   <slot>chooseCollision()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>70</x>
     <y>97</y>
    </hint>
    <hint type="destinationlabel">
     <x>349</x>
     <y>224</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSave_Animation</sender>
   <signal>triggered()</signal>
   <receiver>MMainWindow</receiver>
   <slot>saveAnimation()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>349</x>
     <y>224</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>openNewFile()</slot>
  <slot>saveFile()</slot>
  <slot>selectMaterial(QModelIndex)</slot>
  <slot>chooseMaterial()</slot>
  <slot>chooseCollision()</slot>
  <slot>saveAnimation()</slot>
 </slots>
</ui>
